<launch>
    <arg name="config"      default="euroc_mav" /> <!-- euroc_mav, tum_vi, rpng_aruco -->
    <arg name="dataset1"     default="V1_02_medium" />
    <arg name="path_gt1"     default="$(find ov_data)/$(arg config)/$(arg dataset1).csv" />
    <arg name="dataset2"     default="V2_02_medium" />
    <arg name="path_gt2"     default="$(find ov_data)/$(arg config)/$(arg dataset2).csv" />

    <arg name="bag"         default="/home/junlin/Downloads/$(arg config)/$(arg dataset2).bag" />

    <node name="create_bag" pkg="ov_msckf" type="create_bag" output="screen" clear_params="true" required="true">
        <param name="path_gt1"        type="str" value="$(arg path_gt1)" />
        <param name="path_gt2"        type="str" value="$(arg path_gt2)" />
        <param name="path_bag"        type="str" value="$(arg bag)" />
    </node>
    <node type="rviz" name="rviz" pkg="rviz" args="-d $(find ov_msckf)/launch/display_cse.rviz" />
    
</launch>
